packages/components/src/composite/index.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @see https://reakit.io/docs/composite/
+ */
+export {
+	Composite,
+	CompositeGroup,
+	CompositeItem,
+	useCompositeState,
+} from 'reakit';
```

Comment:
I'd be curious to learn from @diegohaz if he expects any breaking change in the future for these APIs. which would make it harder for us to upgrade "reakit"

Reply:
Yeah! There will be a few breaking changes in v2. Mostly prop names and other small things, but enough to break the backwards compatibility promise on `@wordpress/components`.

Reply:
It's a similar construct to how we consume React. If we can ensure some long-term support or a good migration path then it shouldn't be an issue at all. Those are low-level APIs that most of the Reakit components use as well so we would be in the same boat 😅

Reply:
Potentially,   we could consider this an internal API `__unstable`

Reply:
Yes, it might be a good option for a start until @diegohaz tells us it's a final API :)

Reply:
https://github.com/WordPress/gutenberg/pull/28085/commits/73b6cd3dee8847a018997637b2800af63cb16e6f adds a change that adds unstable `DisclosureContent` component to act as an example.

Reply:
I think we should go for `__unstable` for now and build our own API on top of the composite/disclosure component over time. This way, if something changes in the Reakit API we can migrate our internal usage without affecting the API we expose.

Reply:
Since other packages are using those components I think it makes sense having a `@wordpress/components` own API anyway.

Reply:
I will update the codebase tomorrow. It also means that I will only add code comments explaining our plans for the future. README.md files would get exposed in the developer portal and bring some confusion.